src/manage/getAccessRequestFromRedirectUrl.test.ts

```diff
@@ -155,7 +155,7 @@ describe("getAccessRequestFromRedirectUrl", () => {
     );
 
     const { accessRequest, requestorRedirectUrl } =
-      await getAccessRequestFromRedirectUrl(redirectedToUrl.href);
+      await getAccessRequestFromRedirectUrl(redirectedToUrl);
```

Comment:
@ThisIsMissEm that's a bit far fetched, but this actually does add coverage for using an URL object instead of a plain string. However I think you mentioning it had to be done indicates that this is too implicit, which I had kind of a feeling when doing it in the first place. There is some setup in the tests I didn't want to duplicate, but I'll see how I can make this cleaner.

Reply:
I think maybe you could just do:

```
const accessRequestFromString = await getAccessRequestFromRedirectUrl(redirectedToUrl.href);
const accessRequestFromUrl = await getAccessRequestFromRedirectUrl(redirectedToUrl);

expect(accessRequestFromString.accessRequest).toStrictEqual(accessRequestFromUrl.accessRequest);
expect(accessRequestFromString.requestorRedirectUrl).toBe(accessRequestFromUrl.requestorRedirectUrl);

const { accessRequest, requestorRedirectUrl } = accessRequestFromUrl;
```